remotes/promises.go

```diff
@@ -211,3 +211,8 @@ func (s *errgroupScheduler) schedule(do func(ctx context.Context) error) promise
 func (s *errgroupScheduler) ctx() context.Context {
 	return s.context
 }
+
+// nolint: unparam
```

Comment:
what param was this complaining about?

Reply:
it is the return param. it is unexported and only used in a defer statement for now...